zephyrdoom/src/main.c

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ff.h>
+#include <hal/nrf_gpio.h>
+#include <nrfx_clock.h>
+#include <string.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>z
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/fs/fs.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sd/sd.h>
+#include <zephyr/storage/disk_access.h>
+
+#include "deh_str.h"
+
+LOG_MODULE_REGISTER(sd_card, LOG_LEVEL_DBG);
```

Comment:
Please change the "sd_card" to something like "doom_main", also don't hardcode the log level. Instead of LOG_LEVEL_DBG, put there LOG_LEVEL_<name_of_the_module>_LOG_LEVEL and then put LOG_LEVEL_<name_of_the_module>_LOG_LEVEL_DBG=y to the prj.conf